### PR TITLE
Add option to clone newdles

### PR DESCRIPTION
**Pull request by tomasr8**
Adds a simple form where you can choose to keep participants and/or timeslots. You can also move the timeslots to some other date, that is, the timeslots will start on the selected date. This form then takes you to a prefilled newdle creation page where you can proceed as usual when creating a newdle.

image
image
image

### CODE REVIEW

File: newdle/client/src/components/creation/ClonePage.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                  />
+                </div>
+              )}
+            </div>
+          )}
+        </div>
+      </div>
+      <div className={styles['create-button']}>
+        <Button color="violet" type="submit" disabled={!canSubmit} onClick={cloneNewdle}>
+          <Trans>Review newdle</Trans> 🍜
```
Comment (ThiefMaster) [inline newdle/client/src/components/creation/ClonePage.js:138]: hm... "Review cloned newdle"? and probably worth showing some message on the next page that the new-dle hasn't been saved yet so people do not think they are already done? but we could also see if people realize it on their own first

File: newdle/client/src/components/summary/SummaryPage.js [hunk 2]
```diff
@@ -211,11 +212,10 @@ export default function SummaryPage() {
           {isCreator && (
             <div className={styles['button-row']}>
               {gridViewActive && actions}
-              <Button
-                color="red"
-                className={styles['delete-button']}
```
Comment (ThiefMaster) [inline newdle/client/src/components/summary/SummaryPage.js:216]: Was this removed on purpose?

Comment (tomasr8) [reply newdle/client/src/components/summary/SummaryPage.js:216]: I just removed the class (which doesn't exist)

File: newdle/client/src/components/creation/ClonePage.js (hunk 1 above)
Comment (tomasr8) [reply newdle/client/src/components/creation/ClonePage.js]: I added an info message to the first step

**Review by ThiefMaster — APPROVED** [review]

**Pull request merged**